From 1d2913d9b9d6bf3a55d319b39b36aef37a459f71 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 13 Sep 2019 06:40:03 +0200
Subject: [PATCH 28/43] shaped-texture: Declare that we inherit from GObject

As per commit 75cffd0ec MetaShapedTexture is a GObject implementing the Clutter
Content interface.

Reflect this on the type declaration.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/796
---
 src/meta/meta-shaped-texture.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meta/meta-shaped-texture.h b/src/meta/meta-shaped-texture.h
index d38a6dae1..e0a8207ab 100644
--- a/src/meta/meta-shaped-texture.h
+++ b/src/meta/meta-shaped-texture.h
@@ -37,7 +37,7 @@ META_EXPORT
 G_DECLARE_FINAL_TYPE (MetaShapedTexture,
                       meta_shaped_texture,
                       META, SHAPED_TEXTURE,
-                      ClutterActor)
+                      GObject)
 
 
 META_EXPORT
-- 
2.23.0

